eZ/Bundle/EzPublishCoreBundle/Command/DeleteUsersFromTrashCommand.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * File containing the DeleteUsersFromTrashCommand class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use eZ\Publish\Core\Repository\Values\ContentType\ContentType;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use eZ\Publish\API\Repository\Values\Content\Query;
+
+class DeleteUsersFromTrashCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('ezpublish:deleteusersfromtrash')
+            ->setDescription('Deletes all users from trash, See https://jira.ez.no/browse/EZP-25643');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        /** @var $repository \eZ\Publish\API\Repository\Repository */
+        $repository = $this->getContainer()->get('ezpublish.api.repository');
+        $trashService = $repository->getTrashService();
+        $contentTypeService = $repository->getContentTypeService();
+
+        $repository->setCurrentUser($repository->getUserService()->loadUser(14));
+
+        $query = new Query();
+        $query->filter = null;
```

Comment:
Did you consider using the Query filter to filter on user content types ? You could first check every content type, to see which ones have a user field definition, and filter on those. It would avoid loading the content types in the loop below.